libs/wingsdk/src/cloud/domain.ts

```diff
@@ -0,0 +1,61 @@
+import { Construct } from "constructs";
+import { fqnForType } from "../constants";
+import { App } from "../core";
+import { Node, Resource } from "../std";
+
+/**
+ * Global identifier for `Domain`.
+ */
+export const DOMAIN_FQN = fqnForType("cloud.Domain");
+
+/**
+ * Options for `Domain`.
+ */
+export interface DomainProps {
+  /**
+   * The website's custom domain name.
+   * @example "example.com"
+   * @default - a domain is generated by the cloud provider
```

Comment:
What's the use case of defining a Domain without a domain name? Feels like it should be a required field.

Reply:
You're right

change to not optional